packages/calypso-products/src/translations.tsx

```diff
@@ -495,6 +495,276 @@ export const getJetpackProductsFeaturedText = (): Record< string, TranslateResul
 		[ PRODUCT_JETPACK_SOCIAL_BASIC_MONTHLY ]: socialFeaturedText,
 	};
 };
+
+export const getJetpackProductsWhatIsIncluded = (): Record< string, Array< TranslateResult > > => {
+	const realTimeBackup = translate( 'Real-time backups as you edit' );
+
+	const orderBackups = translate( 'WooCommerce order and table backups' );
+	const cloudBackups = translate( 'Redundant cloud backups on our global network' );
+	const prioritySupport = translate( 'Priority support' );
+
+	const backupIncludesInfoT1Storage = translate( '10GB of cloud storage' );
+	const backupIncludesInfoT2Storage = translate( '1TB (1,000GB) of cloud storage' );
+
+	const backupIncludesInfoMonthlyLog = translate( '30-day activity log archive' );
+	const backupIncludesInfoYearlyLog = translate( '1 year activity log archive' );
```

Comment:
@vikneshwar, we have some incorrect data here: 

The way is is defined right now is:  (which is incorrect)

- Backup T1 (monthly) has: '_**30-day activity log archive**_'
- Backup T1 (yearly) has: '_**1 year activity log archive**_' 
- Backup T2 (monthly & yearly) has '_**30-day activity log archive**_'

The way it **should** be:

- Backup T1 (yearly & monthly) should have '_**30-day activity log archive**_'.
- Backup T2 (yearly & monthly) should have '_**1 year activity log archive**_'.

Reply:
Could you create a new PR with the above corrections ^^?